FRPs/FRP-13.md

```diff
@@ -0,0 +1,35 @@
+---
+id: 13
+title: Flashbots in eth2
+team: @obadiaa @tkstankacz @jparyani @fxfactorial
+created: 2021-06-04
+status: pending
+---
+
+# Flashbots in eth2
```

Comment:
same comment as in FRP-12: I'd try to provide a bit more structure to the list in the "plan" section. this looks a bit cleaner to me, but they still feel a tad like loose ideas. that's fine at this stage, but there's clearly some motivation for them--not sure this is clear for the reader from the intro section. to be more concrete: why do we need to look at pools? non-obvious from the intro. I'd favor a spelt-out version as opposed to a list, or some sectioning that provides a bit more motivation for the questions.